app/viewmodels/checkAnswers/memberDetails/MembersLastUKAddressSummary.scala

```diff
@@ -31,7 +31,16 @@ object MembersLastUKAddressSummary {
   def row(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
     answers.get(MembersLastUKAddressPage).map {
       answer =>
-        val value = HtmlFormat.escape(answer.line1).toString + "<br/>" + HtmlFormat.escape(answer.line2).toString
+        val value = Seq(
+          Some(answer.line1),
+          answer.line2,
+          answer.townOrCity,
+          answer.county,
+          answer.postcode
+        ).flatMap {
+          case Some(part) if !part.trim.isEmpty => Some(HtmlFormat.escape(part))
+          case _                                => None
+        }.mkString("<br>")
 
         SummaryListRowViewModel(
           key     = "membersLastUKAddress.checkYourAnswersLabel",
```

Comment:
This is about the route below, If I look up an address which has a city, and I go back to edit that address, the city gets lost in the edit page. I understand there was some discussion about this and the page has been updated but I believe it still doesn't have a city input, instead opting for the address line 4 or 3. This is different to the way that city is handled in the lookup service where it is given it's own json key. We need to think about how to handle this.